modules/mobile-agents/android/client/src/org/wso2/mdm/agent/api/Root.java

```diff
@@ -88,9 +89,9 @@ public boolean checkRootBySuperUserApk() {
 	 */
 	public boolean checkRootBySuAccess() {
 		if (new ShellExecutor().executeCommand(SU_CHECK_COMMAND) != null) {
-            		if (Constants.DEBUG_MODE_ENABLED){
-                		Log.d(TAG, "su command enabled");
-            		}
+			if (Constants.DEBUG_MODE_ENABLED){
```

Comment:
Fixed in Commit [https://github.com/wso2/product-mdm/commit/b0c66506d24fb5288300bec4224f1eaa58c7b426]